db/c.cc

```diff
@@ -1145,6 +1145,34 @@ rocksdb_column_family_handle_t** rocksdb_create_column_families(
   return c_handles;
 }
 
+rocksdb_column_family_handle_t** rocksdb_create_column_families_with_options(
```

Comment:
I know the other APIs don't have method comments, but maybe we could start ? In particular state that this method takes a set of options per CF.

Reply:
Yes, but wasn't be better to add doc comments to c.h file?

Reply:
You are correct, at least in this case where we are documenting the API. Thanks for doing that.